pgsql-flexible-server.tf

```diff
@@ -44,6 +46,16 @@ resource "azurerm_postgresql_flexible_server" "pgsql_server" {
 
   sku_name = var.pgsql_sku
 
+  dynamic "authentication" {
+
+    # Include this block only if var.set_ad_admin is set to a non-null value.
+    for_each = var.set_ad_admin[*]
+    content {
+      active_directory_auth_enabled = true
```

Comment:
I assume that `password_auth_enabled` defaults to `true` and setting this won't disable it?

Docs are unclear:
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/postgresql_flexible_server